Makefile

```diff
@@ -29,11 +29,13 @@ COMMIT_HASH := $(shell git rev-parse HEAD)
 
 GOBUILD := go build -v
 GOINSTALL := go install -v
-GOTEST := go test 
+GOTEST := go test
 
 GOVERSION := $(shell go version | awk '{print $$3}')
 GOFILES_NOVENDOR = $(shell find . -type f -name '*.go' -not -path "./vendor/*" -not -name "*pb.go" -not -name "*pb.gw.go" -not -name "*.pb.json.go")
 
+SUBSERVERS := autopilotrpc chainrpc invoicesrpc neutrinorpc peersrpc routerrpc signrpc walletrpc watchtowerrpc wtclientrpc
```

Comment:
This is already defined as `RPC_TAGS` in `make/testing_flags.mk`. I think we should re-use that variable instead, even if it defines build tags that aren't actually build tags (`routerrpc` and `verrpc` aren't behind build tags, they're always built).

Reply:
Ah, but I see that the falafel script adds build tags to the generated mobile files, so we actually need the additional `routerrpc` and `verrpc` tags from the `RPC_TAG` variable!

Reply:
`routerrpc` was already there, but I've removed the `SUBSERVERS` var in favor of `RPC_TAGS`. I've also added `verrpc` to the falafel script.